src/themes.d.ts

```diff
@@ -0,0 +1,100 @@
+import { createTheme, PaletteColor, PaletteColorOptions, Theme } from "@mui/material/styles";
```

Comment:
Why do we need this file here? What is it doing?

Reply:
to declare custom mui theme types

Reply:
@Ayomikuolatunji until there is a need, please remove this.